tox.ini

```diff
@@ -104,7 +104,8 @@ commands=
     {[common-lint]commands}
     flake8 {toxinidir}/tests --exclude="trinity,p2p"
     # TODO: Drop --ignore-missing-imports once we have type annotations for eth_utils, coincurve and cytoolz
-    mypy --follow-imports=silent --warn-unused-ignores --ignore-missing-imports --no-strict-optional --check-untyped-defs --disallow-incomplete-defs -p eth
+	mypy --follow-imports=silent --warn-unused-ignores --ignore-missing-imports --no-strict-optional --check-untyped-defs --disallow-incomplete-defs --disallow-untyped-defs --disallow-any-generics eth/utils
```

Comment:
@cburgdorf how about just two mypy runs?  one that has the current config and one that we slowly expand to cover only the modules which have been updated.

Reply:
@pipermerriam yes, that's my plan. I just think we do not need to do it for each and every submodule (hence,`eth.tools`, `eth.vm`, then the rest) especially if we consider this whole thing to be finished over the course of the next, say, two weeks.

Reply:
@cburgdorf got it. But since I have started on `eth.utils` first unfortunately, how about we have one mypy for the normal configuration that we see today, one for the `eth.utils` submodule and one for the presently working submodule (`eth.tools` and `eth.vm`)?

Reply:
Cool with me!